src/graph/test/GoTest.cpp

```diff
@@ -406,15 +406,16 @@ TEST_P(GoTest, MULTI_EDGES) {
     }
     {
         cpp2::ExecutionResponse resp;
-        auto *fmt = "GO FROM %ld OVER serve, like REVERSELY YIELD serve._dst, like._dst";
+        auto *fmt = "GO FROM %ld OVER serve, like REVERSELY "
+                    "YIELD serve._dst, like._dst, serve._type, like._type";
         auto &player = players_["Russell Westbrook"];
         auto query = folly::stringPrintf(fmt, player.vid());
         auto code = client_->execute(query, resp);
         ASSERT_EQ(cpp2::ErrorCode::SUCCEEDED, code);
-        std::vector<std::tuple<int64_t, int64_t>> expected = {
-            {0, players_["James Harden"].vid()},
-            {0, players_["Dejounte Murray"].vid()},
-            {0, players_["Paul George"].vid()},
+        std::vector<std::tuple<int64_t, int64_t, int64_t, int64_t>> expected = {
+            {0, players_["James Harden"].vid(), 0, -5},
```

Comment:
I remember back in the day that the `_type` would return a name which is a string. And i think it is not helpful to users if we return an integer. Maybe it is better to return the type name.

Reply:
Good catch! but the type of propName is EdgeType, the type is **INT**,  there are many places will get it, if change the type of it, need to change the type in the schema, it may bring other problems.